ksqldb-common/src/main/java/io/confluent/ksql/config/ImmutableProperties.java

```diff
@@ -35,6 +35,7 @@
       .add(KsqlConfig.KSQL_READONLY_TOPICS_CONFIG)
       .add(KsqlConfig.KSQL_SOURCE_TABLE_MATERIALIZATION_ENABLED)
       .add(KsqlConfig.KSQL_HEADERS_COLUMNS_ENABLED)
+      .add(KsqlConfig.KSQL_SHARED_RUNTIME_ENABLED)
```

Comment:
Will this affect setting via LD?

Reply:
My understanding was this only affects the query level properties but I am not 100% sure. @rodesai do you know perhaps?

Reply:
My understanding is that LD is not via fflag but via external overrides e.g. incoming requests. If yes then they would not be affected.